docs/StardustDocs/topics/addDf.md

```diff
@@ -10,7 +10,7 @@ Returns [`DataFrame`](DataFrame.md) with union of columns from several given [`D
 df.add(df1, df2)
 ```
 
-<dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Modify.addDfs.html"/>
+<dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Modify.addDataFrames.html"/>
```

Comment:
Hm, i don't see this file in the snippets. You forgot to commit it or it wasn't generated?

Reply:
lol, we need an auto-commit for these too